modules/800-deckhouse-tools/images/web/werf.inc.yaml

```diff
@@ -0,0 +1,58 @@
+{{- $version := "v0.3.0" }}
+image: {{ .ModuleName }}/{{ .ImageName }}
+fromImage: common/nginx-static
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+git:
+- add: /{{ $.ModulePath }}modules/800-{{ $.ModuleName }}/images/{{ $.ImageName }}/nginx.conf
+  to: /opt/nginx-static/conf/nginx.conf
+  stageDependencies:
+    setup:
+    - '**/*'
+import:
+  - artifact: {{ .ModuleName }}/{{ .ImageName }}-artifact
+    add: /app
+    to: /app
+    before: setup
+docker:
+  ENTRYPOINT: [ "/opt/nginx-static/sbin/nginx", "-g", "daemon off;" ]
+---
+artifact: {{ .ModuleName }}/{{ .ImageName }}-artifact
+from: {{ .Images.BASE_GOLANG_22_BULLSEYE_DEV }}
+git:
+  - add: /{{ $.ModulePath }}modules/800-{{ $.ModuleName }}/images/{{ $.ImageName }}/static
+    to: /static
+    stageDependencies:
+      setup:
+        - '**/*'
+import:
+  - artifact: {{ .ModuleName }}/{{ .ImageName }}-frontend-artifact
+    add: /app/dist
+    to: /app
+    before: setup
+shell:
+  setup:
+    - export GOPROXY={{ $.GOPROXY }}
+    - git clone --depth 1 --branch {{ $version }} {{ $.SOURCE_REPO }}/deckhouse/deckhouse-cli.git
+    - sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d v3.38.0
```

Comment:
would we like to put the taskfile in the dev image or in the SOURCE_REPO instead of downloading it directly?
This will be useful for the CSE edition

Reply:
Good point.

Reply:
task already in BASE_GOLANG_22_BULLSEYE_DEV